src/types.ts

```diff
@@ -68,3 +69,5 @@ export type Route<E extends Env = Env, N = Node> = {
 }
 
 export type AppRoute<E extends Env = Env> = (app: Hono<E>) => void
+
+export type LayoutContext = Omit<Context | 'render', 'setRenderer'>
```

Comment:
It would lead unexpected behavior to override the rendering in layouts, so omitting them here.